### PR TITLE
[PTQ] Add default batch sizes for PTQ conformance test

### PR DESCRIPTION
**Pull request by kshpv**
### Changes

Add default batch_size for calibration dataset for every model.
If model has no "batch_size" parameter meaning that it doesn't support batch_size > 1

### Reason for changes

Speed up quantization

### Related tickets

N/A

### Tests

N/A

### CODE REVIEW

**Comment by kshpv** [comment]
job number 366 passed

File: tests/post_training/model_scope.py [hunk 1]
```diff
@@ -141,13 +147,15 @@
...
         "backends": ALL_PTQ_BACKENDS,
+        "batch_size": 128,
     },
     {
         "reported_name": "timm/inception_resnet_v2",
         "model_id": "inception_resnet_v2",
         "pipeline_cls": ImageClassificationTimm,
         "compression_params": {},
         "backends": NNCF_PTQ_BACKENDS,
+        "batch_size": 64,
```
Comment (KodiaqQ) [inline tests/post_training/model_scope.py:158]: Why is it 64 and 32 for several models?

File: tests/post_training/data/ptq_reference_data.yaml [hunk 2]
```diff
@@ -19,218 +19,218 @@ timm/crossvit_9_240_backend_CUDA_TORCH:
...
 timm/mobilenetv3_small_050_backend_CUDA_TORCH:
   metric_value: 0.41888
 timm/mobilenetv3_small_050_backend_FP32:
   metric_value: 0.57906
 timm/mobilenetv3_small_050_backend_ONNX:
-  metric_value: 0.42104
+  metric_value: 0.41828
 timm/mobilenetv3_small_050_backend_OV:
-  metric_value: 0.42184
+  metric_value: 0.41874
```
Comment (KodiaqQ) [inline tests/post_training/data/ptq_reference_data.yaml:146]: Shouldn't be investigated such a significant drop?

**Comment by KodiaqQ** [comment]
Are there any numbers that show speed up for the conformance runs?

**Review by KodiaqQ — APPROVED** [review]

**Pull request merged**